wp-content/themes/currentorg/partials/projects-base.php

```diff
@@ -8,6 +8,12 @@
 $args = array(
 	'post_type' => 'projects',
 	'post_status' => 'publish',
+	'orderby' => array(
+		// sort date by DESC so it grabs newest years first
+		'date' => 'DESC',
+		// sort titel by ASC so it grabs alphabetically
+		'title' => 'ASC'
```

Comment:
Some titles start with punctuation like `¿` or `“` or `@`, which sorts to before `a`. Also, titles prefixed with "The" come in the `T` section.

I don't think there's anything we can do about that.

Reply:
I don't think so either unless we want to overwrite the default WP functionality 😬